auditbeat/module/auditd/audit_linux_test.go

```diff
@@ -141,23 +141,24 @@ func TestLoginType(t *testing.T) {
 
 	for idx, expected := range []common.MapStr{
 		{
-			"event.category": "authentication",
-			"event.type":     "authentication_failure",
+			"event.category": []string{"authentication", "user-login"},
+			"event.type":     []string{"start", "authentication_failure"},
```

Comment:
Are these second ones (user-login and auth_failure) marked as deprecated for deletion in 8.0?